correlation.py

```
@@ -321,16 +420,83 @@ def compute_correlation_matrix(
     # Round for display
     corr_matrix_rounded = corr_matrix.round(3)
     
-    # Create Heatmap
-    # Using red-white-blue scale (Red=Positive, Blue=Negative)
+    # Calculate p-values for all pairs
+    n = len(data.dropna())
```

Ruff (0.14.10)

Local variable `n` is assigned to but never used

Remove assignment to unused variable `n`

(F841)